build.gradle

```diff
@@ -92,6 +92,15 @@ configure(allprojects - project(':distribution:archives:integ-test-zip')) {
   }
 }
 
+// TODO what is the right way to use maven local repo?
```

Comment:
No need for that, to use Maven local pass `-Drepos.mavenLocal` to Gradle command line